Sources/Application/MainCoordinator.swift

```diff
@@ -105,6 +109,26 @@ final class MainCoordinator: NSObject,
             return
         }
     }
+    
+    func checkAppVersion() {
```

Comment:
This code will need to move into Joe's new coordinator when that is created and merged, but happy to approve this for now based on that change happening in a later PR.